### ISSUE TITLE
Only 'en' locale is dumped by default

### ISSUE THREAD

**Issue by borNfreee** [willdurand/BazingaJsTranslationBundle#124]
Hi,
I'm not sure this is a bug, may be I don't understand documentation correctly.
My config:

```php
bazinga_js_translation:
    locale_fallback:      en
    default_domain:       messages
```

I have 3 locales on my site: en, ru, be. When I run `php app/console bazinga:js-translation:dump` only en.js/en.json files appear on the folders. At the same time documentation [says](https://github.com/willdurand/BazingaJsTranslationBundle/blob/master/Resources/doc/index.md#active-locales): 
```
By default, all locales are dump. You can define an array of active locales: ...
```

Then when I explicitly write all my locales they are dumped correctly with this config:
```php
bazinga_js_translation:
    locale_fallback:      en
    default_domain:       messages
    active_locales:
        - ru
        - en
        - be
```

Is it a bug?

**Comment by borNfreee** [comment]
bump @willdurand

**Comment by borNfreee** [comment]
fixed in master

**Comment by willdurand** [comment]
I just released [v2.2.0](https://github.com/willdurand/BazingaJsTranslationBundle/releases/tag/v2.2.0).
